lib/solargraph/complex_type/unique_type.rb

```diff
@@ -248,6 +248,10 @@ def can_assign?(api_map, atype)
         out
       end
 
+      def nullable?
+        nil_type?
+      end
```

Comment:
* fix: Ensure #reduce_class_type and #nullable? are in both ComplexType and UniqueType, a bug this PR tickled